group_vars/galaxyservers.yml

```
@@ -356,3 +356,41 @@ gie_proxy_setup_service: systemd
 gie_proxy_port: 4002
 gie_proxy_verbose: true
 gie_proxy_sessions_path: "{{ galaxy_mutable_data_dir }}/interactivetools_map.sqlite"
+
+# RabbitMQ
+rabbitmq_version: 3.8.16-1
+rabbitmq_plugins: rabbitmq_management
+
+rabbitmq_config:
+- rabbit:
+  - tcp_listeners:
+    - "'127.0.0.1'": 5672
+  - ssl_listeners:
+    - "'0.0.0.0'": 5671
+  - ssl_options:
+     - cacertfile: /etc/ssl/certs/fullchain.pem
```

```suggestion
     - cacertfile: /etc/ssl/certs/{{ openssl_domains[0] }}.crt
```

let's see if it complains...